src/content/en/updates/2017/09/picture-in-picture.md

```diff
@@ -0,0 +1,109 @@
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/picture-in-picture.png #}
+{# wf_featured_snippet: Let's discuss about a Web API that would allow websites to create a floating video window over the desktop #}
+{# wf_blink_components: Blink>Media #}
+
+# Picture In Picture (PiP) {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Since April 2017, Chrome for [Android O supports Picture In Picture]. It allows
+users to play a `<video>` element in a small overlay window that isn't blocked
+by other windows, so that they can watch while doing other things.
+
+Here's how it works: Open Chrome, go to a website that contains a video and
+play it fullscreen. From there, press the Home button to go to your Android
+Home Screen and the playing video will automatically transition to Picture In
+Picture. That's all! Pretty cool right?
+
+<figure>
+  <img src="/web/updates/images/2017/09/picture-in-picture-hero.jpg"
+       alt="Android Picture in Picture photo">
+  <figcaption>
+    <b>Figure 1.</b>
+    Android Picture in Picture photo
+  </figcaption>
+</figure>
+
+It is, but... what about desktop? What if the website wants to control that
+experience?
+
+The good news is that a [Picture In Picture Web API] specification is being
+drafted as we speak. This spec aims to allow websites to initiate and control
+this behavior by exposing the following set of properties to the API:
+
+- Notify the website when a video enters and leaves Picture in Picture mode.
+- Allow the website to trigger Picture in Picture on a video element via a user gesture.
+- Allow the website to exit Picture in Picture.
+- Allow the website to check if Picture in Picture can be triggered.
+
+And this is how it could look like:
+
+    <video id="video" src="https://example.com/file.mp4"></video>
+    
+    <button id="pipButton"></button>
+    
+    <script>
+      // Hide button if Picture In Picture is not supported.
+      pipButton.hidden = !document.pictureInPictureEnabled;
+    
+      pipButton.addEventListener('click', function() {
+        // If there is no element in Picture In Picture yet, let's request Picture
+        // In Picture for the video, otherwise leave it.
+        if (!document.pictureInPictureElement) {
+          video.requestPictureInPicture()
+          .catch(error => {
+            // Video failed to enter Picture In Picture mode.
+          });
+        } else {
+          document.exitPictureInPicture()
+          .catch(error => {
+            // Video failed to leave Picture In Picture mode.
+          });
+        }
+      });
+    </script>
+
+Warning: Code above is not implemented by browsers yet.
+
+## Feedback
+
+So what do you think? Please submit your feedback and raise issues in the
+[Picture In Picture WICG repository]. We're eager to hear your thoughts!
+
+## Preventing Android's default PIP behavior
+
+Today, you can prevent video from Android's defaut PiP behavior in Chrome by
```

Comment:
Change 'from' to 'from using'.

Change 'defaut' to 'default'.

Did I miss something? Why do I need to prevent Android's default PiP behavior if I don't have it yet?

Reply:
Done.
Done.

Android's default PiP behaviour will still be on top of the Web API.
Sadly, as the API doesn't exist yet, there is no proper way to say "no". See https://github.com/WICG/picture-in-picture/issues/25